contrib/kind.sh

```diff
@@ -88,6 +88,7 @@ usage() {
     echo "                 [-ho |--hybrid-enabled] [-ii|--install-ingress] [-n4|--no-ipv4]"
     echo "                 [-i6 |--ipv6] [-wk|--num-workers <num>] [-ds|--disable-snat-multiple-gws]"
     echo "                 [-dp |--disable-pkt-mtu-check]"
+    echo "                 [-df |--disable-forwarding-on-interfaces]"
```

Comment:
is `-on-interfaces` needed? I don't think it gives a lot of context in terms of what interfaces we are disabling the forwarding on.
I think we could stick to `--disable-forwarding` since the explanation below should be enough

Reply:
yeah but at least its telling us its forwarding for interfaces that's being done, even if not providing details of what all interferces. If `--disable-forwarding` is enough I am good with that, @trozet are you ok with `disable-forwarding`?

Reply:
I will move to making this --disable-forwarding as per reviewer suggestion.